liberty-managed/src/test/java/io/openliberty/arquillian/managed/WLPIntegrationClientTestCase.java

```diff
@@ -40,7 +41,7 @@
    public static EnterpriseArchive createDeployment() 
    {
       return ShrinkWrap.create(EnterpriseArchive.class, "test.ear")
-            .addAsModule(ShrinkWrap.create(WebArchive.class, "test.war")
+            .addAsModule(ShrinkWrap.create(WebArchive.class, "test1.war")
```

Comment:
I didn't see a change anywhere else that triggered this. Why did this change?

Reply:
There is an occasional name conflict error when the `war` within the `ear` have the same name, and are executed along with the existing tests. Changing the name of the war ensures this will not happen.